packages/foam-vscode/src/features/janitor.ts

```diff
@@ -69,7 +70,7 @@ async function janitor(foam: Foam) {
 async function runJanitor(foam: Foam) {
   const notes: Resource[] = foam.workspace
     .list()
-    .filter(r => URI.isMarkdownFile(r.uri));
+    .filter(r => getExtension(r.uri.path) === '.md');
```

Comment:
I still believe we should have a `isMarkdownFile` function, so that application code doesn't need to know what extension(s) we use (e.g. we might support `.mdx`, `.markdown`, .. in the future)

Reply:
Ok

Reply:
I've moved this into a `isMarkdown` predicate but the current implementation just checks for `.md`. If you want me to add checks for other extensions just let me know.